packages/blockchain-wallet-v4-frontend/src/modals/Swap/NoHoldings/index.tsx

```diff
@@ -0,0 +1,218 @@
+import { Props as BaseProps } from '../index'
+import { Border, IconBackground, TopText } from '../components'
+import { FormattedMessage } from 'react-intl'
+import { Icon, Text } from 'blockchain-info-components'
+import React from 'react'
+
+import { bindActionCreators, compose } from 'redux'
+import { connect, ConnectedProps } from 'react-redux'
+
+import { actions } from 'data'
+import { FlyoutWrapper } from 'components/Flyout'
+import { SuccessCartridge } from 'components/Cartridge'
+import styled, { css } from 'styled-components'
```

Comment:
order import 
group 1 -> 3rd party imports
group 2 -> aliased imports
group 3 -> relative import

Reply:
Shouldn't we have an eslint rule for that? https://eslint.org/docs/rules/sort-imports